docs/versionsdescription.rst

```diff
@@ -1,44 +1,81 @@
      - Exactly one lepton with :math:`p_T>25` GeV; at least four jets with :math:`p_T>25` GeV; at least one jet with :math:`b`-tag > 0.5
+     - :math:`t\bar{t}` sample variations, ``pt_scale`` variations, ``pt_res`` variations, ``btag`` variations, **W + jets** scale variations, luminosity
      - None
-     - 
-   * - 2.0.0
+   * - 2
      - CMS 2015 Open Data (NanoAOD)
-     - 
-     - 
+     - Exactly one lepton with :math:`p_T>30` GeV; at least four jets with :math:`p_T>30` GeV; at least one jet with :math:`b`-tag > 0.5 (see :ref:`versions-cuts` for additional cuts)
+     - :math:`t\bar{t}` sample variations, ``pt_scale`` variations, ``pt_res`` variations, ``btag`` variations, **W + jets** scale variations, luminosity
      - BDT to predict jet-parton assignment in :math:`t\bar{t}` events
-     - 
+     
+     
+Description of AGC Tags
+================================
+The below table gives a brief overview of the AGC tags in the main repository (<https://github.com/iris-hep/analysis-grand-challenge/tags>`_). The above table describes differences in task, while the below table describes differences in implementation
+
+.. list-table:: Tags
+   :widths: 9 5 5 27 27 27 
+   :header-rows: 1
+   
+   * - Tag
+     - Version
+     - Released
+     - Available Pipelines
+     - Systematics
+     - Dependency Management
+   * - 0.1.0
+     - 0
+     - Yes
+     - Pure ``coffea``; ``coffea`` with ``ServiceX`` processor
+     - Systematic variations within ``coffea`` processor are manually calculated using ``awkward`` array logic (jet :math:`p_T` variations are not propagated through signal region observable calculation)
+     - Functions used in ``coffea`` processor are defined in the notebook
+   * - 0.2.0
+     - 0
+     - Yes
+     - Pure ``coffea``; create cached files using ``ServiceX`` queries followed by standalone ``coffea`` processing
+     - Systematic variations within ``coffea`` processor are manually calculated using ``awkward`` array logic (jet :math:`p_T` variations are not propagated through signal region observable calculation)
+     - Functions used in ``coffea`` processor are defined in the notebook
+   * - 1.0.0
+     - 1
+     - Yes
+     - Pure ``coffea``; create cached files using ``ServiceX`` queries followed by standalone ``coffea`` processing
+     - Systematic variations within ``coffea`` processor are manually calculated using ``awkward`` array logic (jet :math:`p_T` variations are not propagated through signal region observable calculation)
+     - Functions used in ``coffea`` processor are defined in the notebook
+   * - 1.1.0
+     - 1
+     - Yes
+     - Pure ``coffea``; create cached files using ``ServiceX`` queries followed by standalone ``coffea`` processing
+     - Systematic variations within ``coffea`` processor are manually calculated using ``awkward`` array logic (jet :math:`p_T` variations corrected)
+     - Functions used in ``coffea`` processor are defined in the notebook
+   * - 2.0.0
```

Comment:
here also a "(WIP)"

Reply:
I think that allows skipping the "released" as well, as this is a fairly unique situation now and we'd usually not be talking about unreleased versions